tests/ui/manual_assert.edition2018.stderr

```diff
@@ -77,19 +77,25 @@ LL | |         panic!("panic with comment") // comment after `panic!`
 LL | |     }
    | |_____^
    |
-help: try instead
+help: replace `if`-then-`panic!` with `assert!`
```

Comment:
@y21 here's the change discussed _in action_ with the comment preservation in the UI output